CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ### Fixed
 - Fix FP in CT_CONSTRUCTOR_THROW when the finalizer does not run, since the exception is thrown before java.lang.Object's constructor exits for checked exceptions ([#2710](https://github.com/spotbugs/spotbugs/issues/2710))
+- Fix possible null value in taxonomies of SARIF output ([#2744](https://sarifweb.azurewebsites.net/Validation))
```

Comment:
Can you please modify the link to refer to the issue? Otherwise it would be a bit misleading. You can refer the sarif site as well, if you would like to.

Reply:
Ah yes, thanks for pointing that out. That was not intended.